### ISSUE TITLE
error with vue-cli, webpack and mint-ui@next

### ISSUE THREAD

**Issue by ritksm** [ElemeFE/mint-ui#235]
maybe related to [this](http://stackoverflow.com/questions/33505992/babel-6-changes-how-it-exports-default)?

reproduce steps:
1. vue init webpack demo
2. cnpm install
3. cnpm install mint-ui@next -S
4. cnpm install babel-plugin-component -D

then edit .babelrc, add:
```
{
  "plugins": ["other-plugin", ["component", [
    { "libraryName": "mint-ui", "style": true }
  ]]]
}
```
then edit App.vue, add:

```
import Vue from 'vue';
import { Cell, Button } from 'mint-ui';

Vue.component(Cell.name, Cell);
Vue.component(Button.name, Button);
```

finally: `npm run dev`

then errors will present with:

```
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined(anonymous function) @ vue.common.js?e6cb:39isReservedTag @ vue.common.js?e6cb:3514Vue.(anonymous function) @ vue.common.js?e6cb:3237(anonymous function) @ App.vue?c1df:31(anonymous function) @ app.js:840__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ App.vue?877c:7(anonymous function) @ app.js:852__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ main.js?3479:4(anonymous function) @ app.js:834__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ app.js:587__webpack_require__ @ app.js:556(anonymous function) @ app.js:579(anonymous function) @ app.js:582
```

Then after changing `Vue.component(Cell.name, Cell);` to `Vue.component(Cell.default.name, Cell);` in `App.vue`, the error is gone

and use of Indicator will cause a error like this:

```
App.vue?c555:36Uncaught TypeError: _indicator2.default.start is not a function(anonymous function) @ App.vue?c555:36(anonymous function) @ app.js:840__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ App.vue?877c:7(anonymous function) @ app.js:852__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ main.js?3479:4(anonymous function) @ app.js:834__webpack_require__ @ app.js:556fn @ app.js:87(anonymous function) @ app.js:587__webpack_require__ @ app.js:556(anonymous function) @ app.js:579(anonymous function) @ app.js:582
```

demo files:

[hs.zip](https://github.com/ElemeFE/mint-ui/files/504345/hs.zip)

**Comment by QingWei-Li** [comment]
fixed in 2.0.0-rc.2

**Comment by ritksm** [comment]
2.0.0-rc.2 works
